getting-started-2.0.md

```diff
@@ -0,0 +1,153 @@
+1. Click the _Commit changes_ button.
+1. Go to `https://<my-repo>-<my-name>.project-helix.page/` to admire your work. Replace `<my-repo>` and `<my-name>` with your actual repository and username:  
+![](./dummy.png "TODO: Helix Pages rendering your content")
+1. **Congratulations!** 🎉 You have just published your first page using _Helix Pages_!
+
+> **Hint:** You can add as many Markdown files as you like, and link them in Markdown.
+
+> **Note:** If your GitHub username contains one or more dashes, add an extra dash between the repository and username in the _Helix Pages_ URL. For example, if your user is called `meet-me`, and your repository is `test`, the corresponding _Helix Pages_ URL would be `https://test--meet-me.project-helix.page/`.
+
+## Step 2: Customize header and footer
+
+_Helix Pages_ can render a header and footer based on Markdown content in your GitHub repository:
+
+1. Add a file called `header.md` to your GitHub repository.  
+![](./dummy.png "TODO: Create a header.md")
+1. Add a logo, and maybe a list of other links you would like to show in the header navigation. Here's some [inspiration](https://raw.githubusercontent.com/adobe/helix-example/master/header.md "example header.md").
+1. Click the _Commit changes_ button.
+1. If you added a logo in step 2, upload that now.  
+![](./dummy.png "TODO: Upload logo")
+1. Add a file called `footer.md` to your GitHub repository.
+1. Add some content you would like to be shown in the footer. Here's a [proposal](https://raw.githubusercontent.com/adobe/helix-example/master/footer.md "example footer.md").
+1. Click the _Commit changes_ button.
+1. Reload `https://<my-repo>-<my-name>.project-helix.page/` to see header and footer in action.  
+![TODO: Helix Pages rendering your header and footer](./dummy.png)
+
+## Step 3: Add your own HTML files
+
+> You'll need:
+> - basic HTML knowledge
+
+If you have existing HTML files or snippets that you wish to reuse, you can simply add them to your GitHub repository and have _Helix_Pages_ merge and deliver them for you:
+
+1. Add a `about.html` file to your GitHub repository. The file name can be anything, just make sure it has a `.html` extension.
+1. Add your HTML. Maybe [like so](https://raw.githubusercontent.com/adobe/helix-example/master/htdocs/about.html "example HTML file")?
+1. Click _Commit changes_.
+1. Refresh `https://<my-repo>-<my-name>.project-helix.page/about.html` to see the rendered result.
+
+> **Note:** If there is a `.md` and `.html` with the same name, _Helix Pages_ will render the HTML file.
+
+> **Hint:** If you add a `header.html` or `footer.html`, _Helix Pages_ will embed them for you.
+
+## Add your own styling
+
+> You'll need:
+> - basic CSS knowledge
+
+_Helix Pages_ allows you to override the default CSS. You can either do this directly in GitHub, or start developing locally.
+
+### GitHub
+
+1. Add a `style.css` file to your GitHub repository.
```

Comment:
Does this have to go into `htdocs`?

Reply:
No: I thought the idea was to overlay from the "content" repo, while `htdocs` is something for the "code" repo.